src/app/public/modules/toolbar/toolbar-section.component.html

```diff
@@ -1,3 +1,6 @@
 <div class="sky-toolbar-section">
-  <ng-content></ng-content>
+  <div class="sky-toolbar-section-items">
+    <ng-content></ng-content>
+  </div>
+  <ng-content select="sky-toolbar-view-actions"></ng-content>
```

Comment:
Before, we're including all inner content, but now we're only injecting specific components. If a consumer had custom HTML in their toolbar section, it wouldn't appear, correct?

Reply:
Incorrect - the `ng-content` without the `select` will get everything that isn't pulled out via those that have the `select`. So any custom HTML will go into the `sky-toolbar-section-items` area which is where it would have previously gone just with an extra wrapper so that the flex properties work correctly.

Reply:
Oh, duh. I misread the first `ng-content`. All good.